<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RSI1535" ignored="0" name="Default test(failed)" tests="2" failures="1" timestamp="09 Feb 2020 04:14:03 GMT" time="152.493" errors="0">
  <testcase name="cadastro1Valido" time="31.887" classname="br.com.rsinet.mobile.appium.testes.TesteCadastro">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at br.com.rsinet.mobile.appium.testes.TesteCadastro.cadastro1Valido(TesteCadastro.java:76)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- cadastro1Valido -->
  <testcase name="cadastro2Invalido" time="30.721" classname="br.com.rsinet.mobile.appium.testes.TesteCadastro"/>
</testsuite> <!-- Default test(failed) -->
